main.tex

```diff
@@ -202,18 +202,27 @@ \subsection{Astropy development model}
 documentation.\footnote{\emph{How to make a code contribution}, \url{http://docs.
 astropy.org/en/stable/development/workflow/development_workflow.html}}
 
+\begin{figure}
+\includegraphics[width=\textwidth]{ncommits.pdf}
+\caption{%
+    \emph{Left panel}: Distribution of number of commits per committer.
+    \emph{Right panel}: Cumulative number of commits to the \astropypkg core
+    package over time.
+    \label{fig:ncommits}
+}
+\end{figure}
+
 As of version 2.0, \astropypkg contains $212244$ lines of code\footnote{This
 line count includes comments, as these are often as important for
 maintainability as the code itself.  Without comments there are $142197$ lines
 of code.} contributed by $232$ unique contributors over $19270$ \texttt{git}
 commits.
-The commit distribution \inlinecomment{Tom R}{too vague, maybe 'the distribution of number of commits as a function of ...'} follows an approximate power-law (with log-slope of
-$\approx -0.47$) \inlinecomment{AG}{round to -0.5?  This is astronomy, after all.}, meaning most of the commits were and are contributed by a
-relatively small fraction of the contributor pool.
-Roughly half of the commits were contributed by the top $4$
-contributors, while $\sim 90$ percent of the commits came from the top $24$
-contributors.
-
+\figurename~\ref{fig:ncommits}, left, shows the distribution of total number of
+commits per contributor as of November 2017; from this, it is clear that most
+commits were and are contributed by a relatively small fraction of the
+contributor pool.
```

Comment:
Actually this isn't that **clear** to me (it would only be clear if we showed a cumulative distribution that showed that we hit e.g. 50% after 4 commits). But based on the power law, I don't think it's that obvious, right? I also don't know if that kind of statement is helpful community-wise - to me the plot shows that there is a wide range of contributor levels, with a healthy number of people contributing e.g. >100 commits. This is not a project with 2 core developers doing 99% of the work and 200 people doing 3 commits. That to me is more interesting. Some of the 10-commit contributors have made important changes and we don't want to make it sound like only a few people did most of the work. Concrete suggestion, change 'from this...' to:

*This distribution shows that the core package has been developed by a broad contributor base, with a core team of developers having added over 1000 commits to the repository, and a smooth distribution of contributors all the way down to single commit contributors. In this sense, the development of the core package has been a true community effort and is not dominated by a single individual. It is also important to note that the number of commit is only a rough metric of contribution, as a single commit could be a critical fix in the package or a fix for a typographical error.*

Reply:
Love it